src/net/Mybuild

```diff
@@ -1,25 +1,27 @@
 package embox.net
 
 module dev {
-	option number netdev_quantity=8
```

Comment:
Please, don't touch a code if it's not necessary.
You swap some lines in this commit, can you rollback them?